shopinvader/controllers/main.py

```diff
@@ -20,11 +20,11 @@ class InvaderController(main.RestController):
     _collection_name = "shopinvader.backend"
     _default_auth = "api_key"
 
-    @route(["/shopinvader/<service>/<int:_id>/download"], methods=["GET"])
+    @route(["/shopinvader/<service>/<int:_id>/download"], methods=["GET"], auth=_default_auth)
```

Comment:
Yes, I'm wondering why adding this as main.RestController is aware of the _default_auth...

Reply:
bc you in this case, you declare your own route not managed by the RestController